persistence/src/commonMain/db_user/com/wire/kalium/persistence/Calls.sq

```diff
@@ -100,3 +100,18 @@ SET status = 'CLOSED'
 WHERE
     status != 'CLOSED'
     AND status != 'MISSED';
+
+selectLastCallIfActiveByConversationId:
+SELECT *
+FROM Call
+WHERE -- consider only the last call and return it only if it has one of the active statuses
+    id = (
+            SELECT id
+            FROM Call
+            WHERE conversation_id = :conversationId
+            ORDER BY created_at DESC
+            LIMIT 1
+        )
+    AND conversation_id = :conversationId
+    AND status IN ('STARTED', 'INCOMING', 'ANSWERED', 'ESTABLISHED', 'STILL_ONGOING')
+LIMIT 1;
```

Comment:
Question: I'm wondering if this query is the same as just matching the `conversationId` and `status` filter in the outer query 🤔 

```sql
SELECT *
FROM Call
WHERE conversation_id = :conversationId
  AND status IN ('STARTED', 'INCOMING', 'ANSWERED', 'ESTABLISHED', 'STILL_ONGOING')
ORDER BY created_at DESC
LIMIT 1;
```

Reply:
It's not exactly the same because I wasn't sure whether it's possible to for instance have last call be "not active" but previous one be still "active" so I wanted to make sure that this query takes last call for the given conversation id and then checks **if** it has "active" state, but I see that we usually just update the state of the last call so I guess it's safe to assume that only the last call can be "active" and simplify the query as you proposed. 😃